home/.zshenv

```diff
@@ -0,0 +1,6 @@
+export PYENV_ROOT=~/.pyenv
```

Comment:
I seem to recall there are reasons to prefer this.  (Haven't looked this up recently.)

```suggestion
export PYENV_ROOT=$HOME/.pyenv
```